### DISCUSSION TITLE
Used wrong command

### DISCUSSION THREAD

**Discussion by mrwebbazz** [community/community#23834] [New to GitHub]
<p>Unfortunately, I used the wrong command like “$ git remote add origin <a href="https://github.com/mrwebbazz/Simple-eCommerce.gi" rel="noopener nofollow ugc">https://github.com/mrwebbazz/Simple-eCommerce.gi</a>” look at the end of the command line. exactly I mistake to type the last letter of t.<br>
as a result I can do nothing. i don’t know much more of git.<br>
Please help me.<br>
The output is “remote: Repository not found.<br>
fatal: repository ‘<a href="https://github.com/mrwebbazz/Simple-eCommerce.gi/" rel="noopener nofollow ugc">https://github.com/mrwebbazz/Simple-eCommerce.gi/</a>’ not found”</p>
<p>Please help me…</p>

**Comment by airtower-luna** [comment]
<p>Use <a href="https://git-scm.com/docs/git-remote#Documentation/git-remote.txt-emset-urlem"><code>git remote set-url</code></a> to set the correct URL, e.g.:</p>
<pre><code class="lang-nohighlight">git remote set-url origin https://github.com/mrwebbazz/Simple-eCommerce.git
</code></pre>

**Comment by mrwebbazz** [comment, marked as answer]
<p>Thank You. The problem had been solved.</p>
